App.js

```diff
@@ -1,25 +1,52 @@
-import React from 'react';
-import styled from 'styled-components/native';
-import { SensorComponent } from './components/SensorComponent';
+import React from 'react'
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import HomeScreen from './components/HomeScreen';
+import AffirmationMessage from './components/AffirmationMessage'
+import Joke from './components/Joke'
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`;
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`;
+const Stack = createStackNavigator();
 
 const App = () => {
-  return (
-    <Container>
-      <SensorComponent></SensorComponent>
-    </Container>
-  );
-};
+  
+  return(
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Start" component={HomeScreen}
+        options={{
+          headerTitleStyle: {
+            color: 'thistle',
+          },
+          headerStyle: {
+            backgroundColor: '#833471',
+          },
+        }} />
```

Comment:
You have 3 components, 9 lines each, and the only difference between them is 2 words. This is a great opportunity to create an array containing objects with these 2 fields for everyone option and build this logic with the map().